src/GitTools.Core/GitTools.Core.NETSTANDARD/project.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "LibGit2Sharp-dotnet": "1.0.0",
+    "LibGit2Sharp.NativeBinaries-dotnet": "[1.0.137]",
```

Comment:
Hey, I don't want to fork the Ecosystem, this has downstream implications so we can't merge this until these libraries natively add netstandard support. Sorry